src/button.cpp

```diff
@@ -31,3 +23,16 @@ bool Button::isClicked(sf::Event event){
 	return true;
 	return false;
 }
+void Button::setFont(sf::Font &font){
+	text.setFont(font);
+}
+void Button::setColors(sf::Color backgroundColor, sf::Color textColor){
```

Comment:
We dont know what is background color and what is text color. I think we should make two separate functions: `setBackgoundColor()` and `setTextColor`